uisdk/src/androidTest/java/com/karhoo/uisdk/booking/checkout/bookingconfirmation/BookingConfirmationScreenshotTests.kt

```diff
@@ -0,0 +1,313 @@
+package com.karhoo.uisdk.booking.checkout.bookingconfirmation
+
+import android.content.Intent
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import androidx.test.platform.app.InstrumentationRegistry
+import com.github.tomakehurst.wiremock.common.ConsoleNotifier
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration
+import com.github.tomakehurst.wiremock.extension.responsetemplating.ResponseTemplateTransformer
+import com.github.tomakehurst.wiremock.junit.WireMockRule
+import com.karhoo.sdk.api.model.AuthenticationMethod
+import com.karhoo.uisdk.KarhooUISDK
+import com.karhoo.uisdk.R
+import com.karhoo.uisdk.common.ScreenshotTest
+import com.karhoo.uisdk.common.preferences
+import com.karhoo.uisdk.common.serverRobot
+import com.karhoo.uisdk.common.testrunner.UiSDKTestConfig
+import com.karhoo.uisdk.util.TestData
+import com.karhoo.uisdk.util.TestSDKConfig
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.net.HttpURLConnection
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class BookingConfirmationScreenshotTests :
+    ScreenshotTest<MockBookingConfirmationBaseActivity>(MockBookingConfirmationBaseActivity::class.java) {
+
+    /**
+     * In order to generate locally screenshots, you need to use the -Precord argument
+     * gradlew fulldebugexecuteScreenshotTests -Precord -Pandroid.testInstrumentationRunnerArguments.class=com.karhoo.uisdk.quotes.screenshot.QuotesScreenshotTests
+     *
+     * When you want to compare the screenshots after some UI changes, just run
+     * gradlew fulldebugexecuteScreenshotTests -Pandroid.testInstrumentationRunnerArguments.class=com.karhoo.uisdk.quotes.screenshot.QuotesScreenshotTests
```

Comment:
please change the QuotesScreenshotTests to BookingConfirmationScreenshotTests for easy run

Reply:
Fixed